CMakeLists.txt

```diff
@@ -58,3 +61,19 @@ if(LEATHERMAN_ENABLE_TESTING)
 	enable_cpplint()
     endif()
 endif()
+
+# Install the cmake files we need for consumers
+if (LEATHERMAN_INSTALL)
+    set(CMAKE_FILES
+        cmake/cflags.cmake
+        cmake/leatherman.cmake
+        cmake/options.cmake
+        cmake/leatherman_config.cmake
+    )
+    install(FILES ${CMAKE_FILES} DESTINATION "lib/cmake/leatherman/cmake/")
+
+    configure_file(LeathermanConfig.cmake.in "${PROJECT_BINARY_DIR}/LeathermanConfig.cmake" @ONLY)
+    message(STATUS ${PROJECT_BINARY_DIR})
```

Comment:
Debugging message or intentional?  Guessing the former given it's just a path being output.